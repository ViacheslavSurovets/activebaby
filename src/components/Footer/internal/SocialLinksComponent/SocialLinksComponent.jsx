import React from 'react';
import { SocialLinks, SocialsLinksWrapper } from '@components/Footer/styles';
import { FOOTER_SOCIAL_LINKS_DATA } from '@components/Footer/data';
import { InstagramIcon, VkIcon } from '@icons/index';

const SocialLinksComponent = ( props ) => (
  <SocialLinks>
    <SocialsLinksWrapper href={ FOOTER_SOCIAL_LINKS_DATA.url.vk } target='_blank'>
      <VkIcon width='40' height='40' />
    </SocialsLinksWrapper>
    <SocialsLinksWrapper href={ FOOTER_SOCIAL_LINKS_DATA.url.instagram } target='_blank'>
      <InstagramIcon width='40' height='40' />
    </SocialsLinksWrapper>
  </SocialLinks>
);

export default SocialLinksComponent;
