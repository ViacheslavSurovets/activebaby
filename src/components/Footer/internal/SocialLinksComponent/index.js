export { default as SocialLinksComponent } from './SocialLinksComponent';
