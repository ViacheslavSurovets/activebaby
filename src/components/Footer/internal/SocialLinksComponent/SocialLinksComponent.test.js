import React from 'react';
import { shallow } from 'enzyme';
import SocialLinksComponent from './SocialLinksComponent';


describe ( 'SocialLinksComponent', () => {
  it ( 'SocialLinksComponent Snapshot', () => {
    expect ( shallow ( <SocialLinksComponent /> ) ).toMatchSnapshot ();
  } );
} );
