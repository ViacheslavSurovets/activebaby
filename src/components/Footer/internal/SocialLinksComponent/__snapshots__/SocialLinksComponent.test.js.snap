// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialLinksComponent SocialLinksComponent Snapshot 1`] = `ShallowWrapper {}`;
